Frogies-Project/Assets/Scripts/StorySystem/Behaviour/SimpleStoryTrigger.cs

```diff
@@ -8,6 +8,7 @@ public class SimpleStoryTrigger : MonoBehaviour
     {
         [SerializeField] private StoryActor actor;
         [SerializeField] private StoryNode startNode;
+        [SerializeField] private bool autoTrigger;
```

Comment:
I dont really like this name. Maybe EnableMouseTrigger/EnablePointerTrigger

Reply:
this is for trigger at the beginning of the scene automatically without any action from player

Reply:
I can change the name if you have any better suggestions